core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineStrategy.java

```diff
@@ -75,4 +76,6 @@ default String encloseFullTableName(String schema, String table) {
   SelectQuery buildSelectQuery(SelectQuery.Builder builder, int limit);
 
   UpsertQuery buildUpsertQuery(UpsertQuery.Builder builder);
+
+  Driver getDriverClass();
```

Comment:
Added `getDriverClass()` that returns the driver class for the corresponding RDB. (CC: @laysakura)